services/transaction_record.proto

```diff
@@ -152,4 +155,37 @@ message TransactionRecord {
      * This field is populated only when the EVM address is not specified in the related transaction body.
      */
     bytes evm_address = 21;
+
+    /**
+     * A list of pending token airdrops.</br>
+     * Each pending airdrop represents a single requested transfer from a
+     * sending account to a recipient account. These pending transfers are
+     * issued unilaterally by the sending account, and MUST be claimed by the
+     * recipient account before the transfer MAY complete.<br/>
+     * A sender MAY cancel a pending airdrop before it is claimed.<br/>
+     * An airdrop transaction SHALL emit a pending airdrop when the recipient has no
+     * available automatic association slots available or when the recipient
+     * has set `receiver_sig_required`.<br/>
+     */
+    repeated PendingAirdropRecord new_pending_airdrops = 22;
+}
+
+/**
+ * A record of a new pending airdrop.
+ */
+message PendingAirdropRecord {
```

Comment:
Can just name this to `PendingAirdrop`

Reply:
This is the record stream entry, not the pending airdrop itself.

Reply:
Since all the fields inside this doesn't end with `Record` (Eg: TokenAssociation) it seems odd to have `Record` only for this.

Reply:
Unfortunately the name `PendingAirdrop` here caused confusion; so would you have a better name suggestion?